### ISSUE TITLE
Newsletter Intro with Dark Background Image

### ISSUE THREAD

**Issue by iamtakashi** [Automattic/blockpatterns#281]
Imported from Lettre.

Screenshot: ![dotcompatterns wordpress com__p=7587 preview=true](https://user-images.githubusercontent.com/908665/194596173-6e57db13-7e0e-4fdb-bb1f-2b51a9f62e9d.png)

```
<!-- wp:cover {"url":"https://dotcompatterns.files.wordpress.com/2022/10/envelopes.jpeg","id":7589,"dimRatio":70,"overlayColor":"black","minHeight":90,"minHeightUnit":"vh","align":"full"} -->
<div class="wp-block-cover alignfull" style="min-height:90vh"><span aria-hidden="true" class="wp-block-cover__background has-black-background-color has-background-dim-70 has-background-dim"></span><img class="wp-block-cover__image-background wp-image-7589" alt="" src="https://dotcompatterns.files.wordpress.com/2022/10/envelopes.jpeg" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:group {"style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}}},"textColor":"white","layout":{"type":"constrained","contentSize":"500px"}} -->
<div class="wp-block-group has-white-color has-text-color has-link-color"><!-- wp:site-logo {"width":120,"shouldSyncIcon":false,"align":"center","className":"is-style-rounded"} /-->

<!-- wp:site-title {"textAlign":"center","style":{"typography":{"fontStyle":"normal","fontWeight":"700","lineHeight":"1.3"}},"fontSize":"x-large"} /-->

<!-- wp:group {"style":{"spacing":{"blockGap":"40px"}}} -->
<div class="wp-block-group"><!-- wp:site-tagline {"textAlign":"center","fontSize":"small"} /-->

<!-- wp:jetpack/subscriptions {"subscribePlaceholder":"Email Address","buttonBackgroundColor":"white","textColor":"black","borderRadius":50,"borderColor":"white","className":"is-style-compact"} -->
<div class="wp-block-jetpack-subscriptions wp-block-jetpack-subscriptions__supports-newline is-style-compact">[jetpack_subscription_form subscribe_placeholder="Email Address" show_subscribers_total="false" button_on_newline="false" custom_font_size="16px" custom_border_radius="50" custom_border_weight="1" custom_padding="15" custom_spacing="10" submit_button_classes="has-white-border-color has-text-color has-black-color has-background has-white-background-color" email_field_classes="has-white-border-color" show_only_email_and_button="true" success_message="Success! An email was just sent to confirm your subscription. Please find the email now and click &#039;Confirm Follow&#039; to start subscribing."]</div>
<!-- /wp:jetpack/subscriptions --></div>
<!-- /wp:group -->

<!-- wp:spacer -->
<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:paragraph {"align":"center","fontSize":"small"} -->
<p class="has-text-align-center has-small-font-size"><a href="#posts">Read all posts ↓</a></p>
<!-- /wp:paragraph --></div>
<!-- /wp:group --></div></div>
<!-- /wp:cover -->
```

Preview: https://dotcompatterns.wordpress.com/?p=7587&preview=true

cc: @alaczek @onuro

**Comment by alaczek** [comment]
Looking good 👍

**Comment by iamtakashi** [comment]
Published!
